src/main/java/pl/orki/hackathon/webapp/venue/control/VenueService.java

```diff
@@ -11,20 +11,17 @@
 @Service
 public class VenueService {
 
-    private final UserRepository userRepository;
     private final VenueRepository venueRepository;
+    private final UserService userService;
 
-    public VenueService(UserRepository userRepository, VenueRepository venueRepository) {
-        this.userRepository = userRepository;
+    public VenueService(VenueRepository venueRepository, UserService userService) {
         this.venueRepository = venueRepository;
+        this.userService = userService;
     }
 
     @Transactional
-    public Optional<Venue> createVenue(Venue venue, Long userId) {
-        return userRepository.findById(userId)
-                .map(user -> {
-                    venue.setUser(user);
-                    return venueRepository.save(venue);
-                });
+    public Optional<Venue> createVenue(Venue venue) {
+        userService.createUser(venue.getUser());
```

Comment:
Similar to https://github.com/OrkiZMajorki/backend/pull/6/files?file-filters%5B%5D=.graphqls&file-filters%5B%5D=.java&hide-deleted-files=true#r500541099